samples/text_generation.py

```diff
@@ -23,15 +23,19 @@
 class UnitTests(absltest.TestCase):
     def test_text_gen_text_only_prompt(self):
         # [START text_gen_text_only_prompt]
+        import google.generativeai as genai
```

Comment:
We could require the import in each sample.

The trade-off is just signal-to-noise ratio.

If we expect each sample to be runnable with just a copy-paste from the site.. some things become difficult: A sample for `tunedModel.delete` requires that you train a tuned model first, that's a case where the signal to noise ratio is really bad.